egs/chime4/s5_6ch/run.sh

```diff
@@ -43,118 +43,91 @@ set -o pipefail
 chime4_data=`pwd`/../..
 # Otherwise, please specify it, e.g.,
 chime4_data=/db/laputa1/data/processed/public/CHiME4
+chime3_data=/data2/archive/speech-db/original/public/CHiME3
```

Comment:
these defaults are very confusing.  I assume this is for MERL.  You should make this part of the 'case' statement below.